site/docs/types/0/0042-Software-Capabilities.md

```diff
@@ -53,6 +53,11 @@ In addition, it is possible to augment software capabilities with the following
 
 Defines the relationship between a `SoftwareCapability` and the `ITInfrastructure` asset that hosts it.
 
+## ProcessingState
+
+Defines a classification for a `SoftwareCapability`. It stores the last sync timestamp for critical elements.
```

Comment:
The `ProcessingState` can be used to describe additional runtime processing information used by various SoftwareCapabilities.